dashboard/test/factories.rb

```diff
@@ -5,7 +5,6 @@
   factory :course do
     name "MyCourseName"
     properties nil
-    plc_course_id nil
```

Comment:
unrelated to this PR, but we don't need to supply nil values to FactoryGirl unless we want to explicitly override a default in the model with nil. Like 7 can also be removed.